4.5 release page (#1138)
godotengine/godot-website: collections/_release_4_5/entry-general-gui-cascade-content-easily-with-foldablecontainer.md

---
type: entry
section: general
subsection: GUI
rank: 0
importance: 3
anchor: cascade-content-easily-with-foldablecontainer
title: Cascade content easily with `FoldableContainer`
text: |

>   The new [`FoldableContainer`](https://docs.godotengine.org/en/latest/classes/class_foldablecontainer.html) node adds an easy way to display a lot of details at the discretion of the user. This feature is also known under the name of "accordion".

Reviewer: ```suggestion
  The new [`FoldableContainer`](https://docs.godotengine.org/en/latest/classes/class_foldablecontainer.html) node adds an easy way to display a lot of details at the discretion of the user, in an "accordion" view.
```
